nixos-org/webserver.nix

```diff
@@ -154,6 +154,8 @@ in
                 #skins = [ ./wiki-skins ];
                 extraConfig =
                   ''
+                    $wgSiteNotice = "The NixOS wiki is outdated and is being shut down. If you wish to add or improve NixOS documentation, please grab a ticket from the [https://github.com/NixOS/nixpkgs/milestone/8 Move the wiki!] milestone.";
```

Comment:
1/4 of the issues were already closed. i would just point to the milestone, since all that matters is what still needs to be migrated.

i do think that the message can be improved. i never really trust myself to explain stuff correctly :)